testcase_maker/generator.py

```diff
@@ -87,7 +90,8 @@ def generate_stdin(self, override: bool = False):
                 with open(stdin_file, "w", newline="\n") as input_buffer:
                     input_buffer.write(stdin)
                     input_buffer.write("\n")
-                print(f"Saved '{stdin_file}'.")
+                end = timer()
+                print(f"Saved '{stdin_file}'. Took {end-start} seconds .")
```

Comment:
```suggestion
                print(f"Saved '{stdin_file}'. Took {end-start} seconds.")
```
remove extra space here.